src/utils/registration-helpers.ts

```diff
@@ -818,30 +829,40 @@ export const convertToRegistrationSearchItem = (registration: Registration) => {
       ? registration.entityDescription.reference.publicationContext.series
       : undefined;
 
+  const contributors = registration.entityDescription?.contributors.map((contributor) => ({
+    affiliation: contributor.affiliations,
+    correspondingAuthor: contributor.correspondingAuthor,
+    identity: contributor.identity,
+    role: contributor.role.type,
+  }));
```

Comment:
kan legge inn falbck her og fjerne det fra `contributorsPreview` og `contributorsCount`
```suggestion
  })) ?? [];
```